config/locales/es.yml

```diff
@@ -3539,13 +3540,13 @@ es:
         verification_code_label: 'Introduzca el código de 6 números '
         verify: Verifique
       welcome:
-        find_right_service: We’ll ask a few questions, to find the right service for you!
-        help_text: 'We provide multiple tax filing options to households that qualify:'
+        find_right_service: "¡Le haremos algunas preguntas para encontrar el mejor servicio para usted!"
+        help_text: 'Le brindamos opciones múltiples para presentar su declaración de impuestos a hogares que califiquen:'
         list:
-        - "<strong>GetYourRefund</strong> Our certified tax specialists will help you file and collect all tax credits you are owed."
-        - "<strong>File Myself</strong> A quick online option lets you to file and collect tax credits on your own."
-        - "<strong>Express</strong> Our quick online option lets you collect any federal stimulus payments and the Child Tax Credit on your own."
-        title: Welcome to our free service!
+        - "<strong>GetYourRefund</strong> Nuestros especialistas certificados de impuestos le ayudarán a presentar su declaración y a cobrar todos los créditos tributarios que se le deben. "
+        - "<strong>File Myself</strong> Una opción rápida en línea le permite presentar su declaración y cobrar los créditos tributarios por su cuenta.  "
+        - "<strong>Express</strong> Nuestra opción rápida en línea le permite cobrar cualquier pago de estímulo federal y el Crédito Tributario por Hijos por su cuenta. "
```

Comment:
There is an unnecessary ` ` (space) character at the end of this string. It is not a problem, but it is unnecessary.